src/System.Globalization/tests/CompareInfo/CompareInfoTests.cs

```diff
@@ -2,32 +2,87 @@
+        }
+
+        public static IEnumerable<object[]> Equals_TestData()
         {
-            CompareInfo ciENG = CompareInfo.GetCompareInfo("en-US");
-            CompareInfo ciFR = CompareInfo.GetCompareInfo("fr-FR");
+            yield return new object[] { CultureInfo.InvariantCulture.CompareInfo, CultureInfo.InvariantCulture.CompareInfo, true };
+            yield return new object[] { CultureInfo.InvariantCulture.CompareInfo, CompareInfo.GetCompareInfo(""), true };
+            yield return new object[] { new CultureInfo("en-US").CompareInfo, CompareInfo.GetCompareInfo("en-US"), true };
+            yield return new object[] { new CultureInfo("en-US").CompareInfo, CompareInfo.GetCompareInfo("fr-FR"), false };
+            yield return new object[] { new CultureInfo("en-US").CompareInfo, new object(), false };
+        }
 
-            Assert.True(ciENG.Name.Equals("en-US", StringComparison.CurrentCultureIgnoreCase));
-            Assert.NotEqual(ciENG.GetHashCode(), ciFR.GetHashCode());
-            Assert.NotEqual(ciENG, ciFR);
+        [Theory]
+        [MemberData(nameof(Equals_TestData))]
+        public void Equals(CompareInfo compare1, object value, bool expected)
+        {
+            Assert.Equal(expected, compare1.Equals(value));
+            if (value is CompareInfo)
+            {
+                Assert.Equal(expected, compare1.GetHashCode().Equals(value.GetHashCode()));
+            }
         }
 
         [Theory]
-        [InlineData("de-DE", "Ü", "UE", -1)]
-        [InlineData("de-DE_phoneb", "Ü", "UE", 0)]
-        [ActiveIssue(5463, PlatformID.AnyUnix)]
-        public void TestLocaleAlternateSortOrder(string locale, string string1, string string2, int expected)
-        {
-            CultureInfo myTestCulture = new CultureInfo(locale);
-            CompareInfo ci = myTestCulture.CompareInfo;
-            Assert.Equal(expected, ci.Compare(string1, string2));
+        [ActiveIssue(5436, PlatformID.AnyUnix)]
+        [InlineData("abc", CompareOptions.OrdinalIgnoreCase, "ABC", CompareOptions.OrdinalIgnoreCase, true)]
+        [InlineData("abc", CompareOptions.Ordinal, "ABC", CompareOptions.Ordinal, false)]
+        [InlineData("abc", CompareOptions.Ordinal, "abc", CompareOptions.Ordinal, true)]
+        [InlineData("abc", CompareOptions.None, "abc", CompareOptions.None, true)]
+        public void GetHashCode(string source1, CompareOptions options1, string source2, CompareOptions options2, bool expected)
+        {
+            CompareInfo invariantCompare = CultureInfo.InvariantCulture.CompareInfo;
+            Assert.Equal(expected, invariantCompare.GetHashCode(source1, options1).Equals(invariantCompare.GetHashCode(source2, options2)));
+        }
+
+        [Fact]
+        public void GetHashCode_EmptyString()
+        {
+            Assert.Equal(0, CultureInfo.InvariantCulture.CompareInfo.GetHashCode("", CompareOptions.None));
```

Comment:
This was part of a test that was previously marked as ActiveIssue on Unix.  It's causing the build to fail.  I suggest you review the CI logs to see if there are others.